tests/python_tests/test_generate_api.py

```diff
@@ -687,7 +687,7 @@ def test_unicode_pybind_decoding_3():
     model_id, path = 'katuni4ka/tiny-random-phi3', Path('tiny-random-phi3')
     pipe = read_model((model_id, path))[4]
     res_str = []
-    pipe.generate(",", max_new_tokens=4, streamer=lambda x: res_str.append(x))
+    pipe.generate(",", max_new_tokens=4, streamer=lambda x: res_str.extend(list(x)))
```

Comment:
Since with the new streamer `�`  can be spitted out with other characters as well, collect all character into a list instead of collecting list of subwords.